config/prod/xschildw-packertest.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
```

Comment:
there's a new template now, `remote/manage-ec2-linux-v2.j2`.  please use new example template

Reply:
Updated name( and s3 path per sageit review).